.github/workflows/pr-comment.yaml

```diff
@@ -18,7 +18,11 @@ jobs:
           node-version: '16'
```

Comment:
Note that I retained the `github.event.comment.author_association == 'MEMBER' &&` line, felt it is still useful for skipping the entire checkout and install process for what will be the majority of the comments when combined with `contains(github.event.comment.body, 'GET_BUILD')`

cc @reidbarber lets get you added to https://github.com/orgs/adobe/teams/react-spectrum haha